### DISCUSSION TITLE
Greetings! I am facing several issues. I'm willing to help debug my system to catch these bugs...

### DISCUSSION THREAD

**Discussion by GBrunheri** [input-leap/input-leap#2068] [Q&A]
Hello! I hope my message finds you well..

So, as the title says, i am experiencing multiple issues with the latest release (v3.0.2), 
going from the range of: clipboard sharing not working, clipboard sharing making clients/server lose connection, random disconnects every minute or so, lots of "laggy moments" where the mouse just jumps everywhere and is unusable for up to 20 seconds, mouse being stuck at some screens when some conditions are met, etc etc etc... 

So, as you can see, there are a lot of issues, and i don't really know how to debug them all to help you, maintainers, to fix them.
So a little help is appreciated to debug all this, and send you the relevant info.

My setup consists of: 

Server: Windows 11
Client 1: Windows 11
Client 2: Linux Lubuntu

I love Input-Leap, and i hope this project keep evolving..
I am learning to code, and for sure i will come back to this repository and turn into a contributor when i have enough programming skills to help Input-Leap grow and reach its maximum potential!

**Comment by shymega** [comment]
Please open bugs in a set of logically split components based on each issue.
